src/Integration.Vsix.UnitTests/CFamily/CLangAnalyzerTests.cs

```diff
@@ -190,65 +190,6 @@ public async Task TriggerAnalysisAsync_StreamsIssuesFromSubProcessToConsumer()
             statusNotifier.VerifyNoOtherCalls();
         }
 
-        [TestMethod]
-        public async Task TriggerAnalysisAsync_IssuesForInactiveRulesAreNotStreamed()
```

Comment:
Filtering out issues for inactive rules is now the responsibility of the `MessageHandler` class.

There is still a test in this class that tests that issues are streamed.